webapp/components/experiments/ExperimentEditStyle.js

```diff
@@ -0,0 +1,242 @@
+import {
+  bgDarkest,
+  bgLight,
+  bgRegular,
+  secondaryColor,
+  fontColor,
+  radius,
+  primaryColor,
+  experimentInputColor,
+  experimentFieldColor,
+  experimentSvgColor,
+  experimentLabelColor,
+  experimentAutocompleteBorder,
+  errorFieldBorder,
+} from '../../theme';
+
+/**
+ * Edit/Add view of a single Experiment.
+ *
+ * @return {JSX.Element}
+ * @constructor
+ */
+const useStyles = (theme) => ({
+  root: {
+    '& .editExperimentContainer': {
+      '& .editExperimentContent': {
+        overflow: 'auto',
+        maxHeight: 'calc(100vh - 400px)',
+        '& .MuiTypography-body2': {
+          opacity: '0.54',
+        },
+      },
+    },
+    '& .editExperimentBack': {
+      display: 'flex',
+      cursor: 'pointer',
+      paddingLeft: theme.spacing(1),
+      '& .MuiTypography-root': {
+        marginLeft: theme.spacing(1),
+      },
+    },
+    '& .editExperimentBreadcrumb': {
+      display: 'flex',
+      justifyContent: 'space-between',
+      alignItems: 'center',
+      '& .MuiButton-startIcon': {
+        marginRight: theme.spacing(0.4),
+      },
+    },
+    '& .editExperimentAutocomplete': {
+      '& .MuiInputLabel-outlined.MuiInputLabel-shrink': {
+        transform: 'translate(14px, 9px) scale(0.75)',
+      },
+      '& .MuiAutocomplete-input': {
+        padding: '1rem 0.25rem 0.188rem',
+      },
+    },
+    '& .editExperimentList': {
+      display: 'flex',
```

Comment:
Margins, paddings and flexes are better inlined with material-ui boxes (lot of cases below)

Reply:
I just extracted existing code to a separate file, I agree with what you're saying